code/modules/reagents/reagent_containers/food/snacks/cultural/unathi.dm

```diff
@@ -238,3 +239,69 @@
 	icon_state = "chew_koko"
 	item_state = "chew_koko"
 	starts_with = list(/obj/item/clothing/mask/chewable/chewingkoko = 6)
+
+/obj/item/reagent_containers/food/snacks/caramelized_steak_bites
+	name = "caramelized steak bites"
+	desc = "A rich, sticky-sweet dessert for any Unathi. For non-Unathi thixxs may sometimes be considered a main dish, albeit a very sweet one, as far as meat dishes are concerned."
+	icon = 'icons/obj/item/reagent_containers/food/cultural/unathi.dmi'
+	icon_state = "steakbites"
+	trash = /obj/item/trash/bowl_brown
+	reagents_to_add = list(/singleton/reagent/nutriment/protein = 6, /singleton/reagent/drink/caramel = 5)
+	reagent_data = list(/singleton/reagent/nutriment/protein = list("meat" = 5))
+	bitesize = 2
+	filling_color = "#3b1b06"
+
+/obj/item/reagent_containers/food/snacks/caramelized_steak_bites/update_icon()
+	var/percent_steakbites = round((reagents.total_volume / 11) * 100)
```

Comment:
The percentage have to be calculated based on the total amount of reagents this takes, not against a fixed number that might not be updated if the reagents change

Reply:
fixed as requested